bugbug/bugzilla.py

```diff
@@ -282,13 +282,14 @@ def delete_bugs(match):
 def count_bugs(bug_query_params):
     bug_query_params["count_only"] = 1
 
-    r = utils.get_session("bugzilla").get(
-        "https://bugzilla.mozilla.org/rest/bug", params=bug_query_params
-    )
-    r.raise_for_status()
-    count = r.json()["bug_count"]
+    counts = []
+
+    def handler(bug):
+        counts.append(bug["bug_count"])
+
+    Bugzilla(bug_query_params, bughandler=handler).get_data().wait()
```

Comment:
Using `bughandler` expect to receive a response that has the property `bugs`: https://bugzilla.mozilla.org/rest/bug?id=1234567. However, passing the `count_only` returns JSON object that contains `bug_count`: https://bugzilla.mozilla.org/rest/bug?id=1234567&count_only=1

Thus, you need to pass the `Query` class to handle this, example: 

```
Bugzilla(queries=Query(Bugzilla.API_URL, bug_query_params, handler)).wait()
```